From 27b68ff1931a2750a1a3effd6a06414142630867 Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Fri, 11 Nov 2016 20:53:41 +0800
Subject: gdkdisplay-win32.c: Define _WIN32_WINNT

... to be for Vista (0x0600) or later.  This is so that the necessary
items in the Windows headers be activated so that the code will build
properly on mingw-w64, and we already require Vista or later for GTK+.

Thanks Ting-Wei Lan for pointing this out.

See: https://bugzilla.gnome.org/show_bug.cgi?id=768081#c62
---
 gdk/win32/gdkdisplay-win32.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gdk/win32/gdkdisplay-win32.c b/gdk/win32/gdkdisplay-win32.c
index 9267640..ceec5b3 100644
--- a/gdk/win32/gdkdisplay-win32.c
+++ b/gdk/win32/gdkdisplay-win32.c
@@ -17,6 +17,9 @@
  */
 
 #include "config.h"
+
+#define _WIN32_WINNT 0x0600
+
 #include "gdk.h"
 #include "gdkprivate-win32.h"
 #include "gdkdisplay-win32.h"
-- 
cgit v0.12

